examples/test/test_qnn_tooklit.py

```diff
@@ -45,9 +48,11 @@ def _setup_resource(self, use_olive_env):
 
         if use_olive_env:
             os.environ["USE_OLIVE_ENV"] = "1"
-            # prepare model and data
             # retry since it fails randomly
-            run_subprocess(cmd="olive configure-qualcomm-sdk --py_version 3.8 --sdk qnn", check=True)
+            retry_func(
```

Comment:
retry_func calls run_subprocess which can raise subprocess.CalledProcessError. This is not captured in retry_func so it will bail out without any retries.